ansible/roles/kepler_systemd/tasks/metal.yml

```diff
@@ -22,7 +22,7 @@
       ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --rm --sdnotify=conmon -d --replace --name kepler \
           --privileged --network=host --pid=host --rm -e EXPOSE_VM_METRICS="true" -e EXCLUDE_SWAPPER_PROCESS="true" -e \
           ENABLE_PROCESS_METRICS="true" -e EXCLUDE_SWAPPER_PROCESS="true" -v /usr/src:/usr/src -v /sys/:/sys/ -v /proc:/proc -v /etc:/etc \
-          {{ kepler_image }}
```

Comment:
I am more incline to fix this by adding a `/etc/kepler/kepler.config` which contains all the vars value like we do in our [compose manifests](https://github.com/sustainable-computing-io/kepler/tree/main/manifests/compose/default/kepler/etc/kepler/kepler.config). It makes easier to update and track any changes

Reply:
https://github.com/sustainable-computing-io/kepler-metal-ci/actions/workflows/validator_compose.yml seems not a schedule running job? 
btw, we have aws and equinix runner, so to match a specific platform is a long term question.

Reply:
btw, seems the compose validation workflow running without a compose manifests in this repo?

Reply:
@SamYuan1990 The reason I pointed out the compose manifests is that this is also a way in which [Kepler](https://github.com/sustainable-computing-io/kepler/blob/a447077ffd2e40f633f2aa200b1b8d61063573c4/pkg/config/config.go#L240) accepts the config i.e inside `/etc/kepler/kepler.config` like this
```
[root@kepler-4kbs7 /]# ls /etc/kepler/kepler.config/
BIND_ADDRESS    CPU_ARCH_OVERRIDE     ENABLE_GPU              ENABLE_QAT             EXPOSE_HW_COUNTER_METRICS   EXPOSE_KUBELET_METRICS  KEPLER_NAMESPACE  MODEL_CONFIG
CGROUP_METRICS  ENABLE_EBPF_CGROUPID  ENABLE_PROCESS_METRICS  EXPOSE_CGROUP_METRICS  EXPOSE_IRQ_COUNTER_METRICS  KEPLER_LOG_LEVEL        METRIC_PATH
```
So instead of passing this config as env like we are doing above why don't we create these files and specify their respective values? When Kepler is launched it will always see the configs inside `/etc/kepler/kepler.config` and will start as expected. Plus it is also easier to manage/update and change various configs rather than adding one more `-e FOO=bar` to the `ExecStart`

Reply:
> [sustainable-computing-io/kepler-metal-ci/actions/workflows/validator_compose.yml](https://github.com/sustainable-computing-io/kepler-metal-ci/actions/workflows/validator_compose.yml) seems not a schedule running job?
btw, we have aws and equinix runner, so to match a specific platform is a long term question.

The validator compose is an alternative way of deploying Kepler using Docker compose instead of systemd and it is compatible irrespective of any platform I.e aws or equinix 

> btw, seems the compose validation workflow running without a compose manifests in this repo?

No we have an ansible play which pulls the Kepler repo and then use those manifests to deploy kepler and other essential services